pkgs/loom/default.nix

```
@@ -179,6 +180,15 @@ buildPythonPackage {
 
   passthru.loom-cpp = loom-cpp;
 
+  passthru.more_packages = {
+    inherit
+      goftests
+      distributions
+      pymetis
+      parsable
+    ;
+  };
+
```

```suggestion

  passthru.ociImage = callPackage ./ociImage.nix { };
```

So you'd end up with `nix build .#loom.ociImage` rather than a separate sibling package.

---

Interesting. I like the inheritance here.

---

I am weighing this abstraction against like  `.#oci.loom` ...